05-how-to-contribute-to-open-source.md

```diff
@@ -144,10 +144,12 @@ You'll need to have these tools donwloaded and installed on your local machine:
 3. Clone your forked repository to your computer.
 4. Run `npm install` to install the dependencies.
 5. Create a new branch and use a descriptive name related to your contribution — for example, `feat/add-alice`.
-6. Run `npm run contributors: add` in your terminal.
+6. Run `npm run contributors:add` in your terminal.
 
    Follow the instructions to add yourself to the guestbook. After you finish and click enter, you must click enter again to confirm your choices.
 
+   ![Image](./images/cli-tool.gif)
```

Comment:
[improvement]: When it comes to implementing images or GIFs that have a specific purpose, it's best to use a sentence for its alternative text instead of one word. It'll help screen reader users gain a better understanding of this step. 
```suggestion
   ![screenshot of a terminal displaying the steps to creating a profile for the guestbook](./images/cli-tool.gif)
```